datatype/imageblk/write.go

```diff
@@ -161,6 +161,18 @@ func (d *Data) PutVoxels(v dvid.VersionID, vox *Voxels, roiname dvid.InstanceNam
 		extentChanged = true
 	}
 
+	// extract buffer interface if it exists
+	var putbuffer storage.RequestBuffer
+	putbuffer = nil
```

Comment:
You don't need to set to nil, since the zero value for an interface like storage.RequestBuffer is nil.